.github/workflows/trivy.yaml

```diff
@@ -36,20 +44,45 @@ jobs:
         format: 'table'
         severity: "HIGH,CRITICAL"
         output: "trivy-report.txt"
-    
-    - name: Add Trivy Report to PR
-      run: |
-        sudo chown runner:runner trivy-report.txt
-        if [ -s trivy-report.txt ] && [ -n "$(grep -v '^\s*$' trivy-report.txt)" ]; then
-          echo '```' | cat - trivy-report.txt > temp && mv temp trivy-report.txt
-          echo '```' >> trivy-report.txt
-          gh issue comment ${{ github.event.issue.number }} --edit-last -F trivy-report.txt
-        else
-          echo ':star2: No High or Critical CVEs Found :star2:' > trivy-report.txt
-          gh issue comment ${{ github.event.issue.number }} --edit-last -F trivy-report.txt
-        fi
-    
-    - name: Report Failure
-      if: ${{ failure() }}
-      run: |
-        gh issue comment ${{ github.event.issue.number }} --edit-last -b ":x: Trivy scan action failed, check logs :x:"
+    - name: Upload Trivy Report
```

Comment:
Might be good to add a safe `retention-days` period -> https://github.com/actions/upload-artifact?tab=readme-ov-file#inputs.